app/invocation/invocation.css

```diff
@@ -230,3 +230,25 @@
   background: transparent;
   border: 1px solid var(--illustration-color);
 }
+
+.invocation-not-found .titles {
```

Comment:
can we leave this styling the way that it was? the colors padding and border feel a bit off

Reply:
Fair enough - done